client/packages/system/src/Manage/Preferences/api/usePreferencesGraphQL.ts

```diff
@@ -0,0 +1,15 @@
+import {
+  useAuthContext,
+  useGql,
+  useQueryClient,
+} from '@openmsupply-client/common';
+import { getSdk } from './operations.generated';
+
+export const usePreferencesGraphQL = () => {
+  const { client } = useGql();
```

Comment:
Ias looking at this and I think we just need one generic hook for the whole app, used like:

const { sdk, storeId } = useGraphQL(getSdk); <- and this is used inside the hooks